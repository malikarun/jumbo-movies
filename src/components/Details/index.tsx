import React, { Component, Fragment } from 'react';

class Details extends Component {
    render() {
        return (
            <div>
                Details
            </div>
        );
    }
}

export default Details;
