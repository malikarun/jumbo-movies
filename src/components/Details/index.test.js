import React from 'react';
import Details from '.';
import renderer from 'react-test-renderer';
import { BrowserRouter, Route } from 'react-router-dom';

test('Should show an item', () => {
    const match = { params: { id: 1, type: 'tv' } };

    const component = renderer.create(
        <BrowserRouter>
            <Route>
                <Details match={match}></Details>
            </Route>
        </BrowserRouter>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});