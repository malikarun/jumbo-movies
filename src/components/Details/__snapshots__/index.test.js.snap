// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should show an item 1`] = `
<div
  className="Details-root-1"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <button
      className="MuiButtonBase-root-30 MuiButton-root-4 MuiButton-text-6 MuiButton-flat-9"
      disabled={false}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiButton-label-5"
      >
        Home
      </span>
      <span
        className="MuiTouchRipple-root-33"
      />
    </button>
  </a>
</div>
`;
